tests/utils/dockercli/volumelifecycle.go

```diff
@@ -26,17 +26,17 @@ import (
 )
 
 // CreateVolume is going to create vsphere docker volume with given name.
-func CreateVolume(ip, name string) ([]byte, error) {
+func CreateVolume(ip, name, opts string) ([]byte, error) {
```

Comment:
Added a CreateVolumeWithOpts(), a variadic function will need more work in CreateVolume() to parse the args and call the docker command